src/utils/Xhr.js

```diff
@@ -181,6 +195,21 @@ class Xhr {
         };
     }
 
+    /**
+     * Create a default headers object
+     *
+     * @param {DefaultHeaderOptions} options - Used to create default headers
+     */
+    getDefaultHeaders({ language }: DefaultHeaderOptions): StringMap {
```

Comment:
Can this be just part of the `getHeaders` function? Unsure having a default getting makes any difference. Specially considering `getHeaders` also does similar `if () then (`) for headers.

Reply:
Yeah, I guess I can just introduce a new class property for language, then we can remove `this.headers`